assets/css/admin.scss

```diff
@@ -1129,3 +1129,26 @@ h2 .nav-tab-active, nav .nav-tab-active {
 .jm-logo__wrapper:hover .jm-logo__letters {
 	transform: rotateY(90deg);
 }
+
+/**
+ * Adjust upsells styles
+ */
+.wpjm-addon-upsell__badge {
```

Comment:
This needs to be in `menu.scss`, the `admin.scss` styles are only loaded for some of the WPJM pages.
<img width="164" alt="image" src="https://github.com/Automattic/WP-Job-Manager/assets/176949/36cd81cb-14c7-4a5b-a8a5-8c2351825833">

Reply:
Moved in 09d35cc0f57e8138b974482bfe1fdec09e0dc55e. Thanks!

(although I didn't detect any pages that didn't load the styles from `admin.scss` while testing...interesting, and TIL)